pkg/fleet/daemon/local_api.go

```diff
@@ -417,6 +450,37 @@ func (c *localAPIClientImpl) StartExperiment(pkg, version string) error {
 	return nil
 }
 
+// StartInstallerExperiment starts an experiment for a installer package.
+func (c *localAPIClientImpl) StartInstallerExperiment(pkg, version string) error {
+	params := experimentTaskParams{
+		Version: version,
+	}
+	body, err := json.Marshal(params)
+	if err != nil {
+		return err
+	}
+	req, err := http.NewRequest(http.MethodPost, fmt.Sprintf("http://%s/%s/experiment/start-installer", c.addr, pkg), bytes.NewBuffer(body))
```

Comment:
nit: is this the only line that's different from `StartExperiment` ? Could we combine them?